source/administration/console/managing-objects.rst

```diff
@@ -47,6 +48,11 @@ Example actions the user may be able to perform include:
 - Display versions
 - Delete
 
+.. versionadded:: Console v0.24.0
+
+   View the status of uploading or downloading objects with the object manager button available on the top right corner of many Console screens.
```

Comment:
I'd maybe replace "many" with "select" - "many" implies a certain % of screens, which we don't reasonably know. I'm actually going to guess you can probably see it only on the object browser view and subviews, but that's a quibble.

Reply:
Actually, you can see it most everywhere.